index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple Website</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://kit.fontawesome.com/89c0957ca7.js" crossorigin="anonymous"></script>
+
+
+</head>
+<body>
+    <main class="gen-grid">
+     <section class="top">
+        <nav>
+            <ul class="d-flex navbar-elements">
+                <li> <i class="fab fa-apple"> </i> </li>
+                <li>Store</li>
+                <li>Mac</li>
+                <li>iPod</li>
+                <li>iPhone</li>
+                <li>iPad</li>
+                <li>iTune</li>
+                <li>Support</li>
+                <li>
+                     <i id="search-icon" class="fas fa-search"></i>
+                     <input type="text">
+                </li>
+            </ul>
+        </nav>
+        <section class="img-text">
+          <img src = "img/text-1.png" alt="picture1"  >
+          <img src = "img/text-2.png" alt="picture1">
+          <img src = "img/text-3.png" alt="picture1">"
+        </section>
+      </section>
+      <section class="product-grid">
+        <img src = "img/iphone5s.jpg" alt="picture1"> 
+        <img src = "img/iphone5c.jpg" alt="picture1">
+        <img src = "img/happybirthdaymac.jpg" alt="picture1">
+        <img src = "img/supplier.jpg" alt="picture1">
+      </section>
+      <footer class="gen-grid">
+        <p>Shop the <a href="">Apple Online Store</a>(1-800-MY-APPLE), visit an <a href="">Apple Retail Store</a>, or find a <a href="">reseller</a>.</p>
+        <p> <a href="">Site Map</a> <a href="">Hot News</a> <a href="">RSS Feeds</a> <a href="">Media Info</a> <a href="">Enviroment</a> <a href="">Job Opportunities</a> <a href="">Contact Us</a> <img src="img/language.png" alt="usa-flag"> </p>
+        <p>Copyright © 2014 Apple Inc. All rights reserved.  <a href="">Terms of Use</a>  <a href="">Privacy Policy</a></p>
+      </footer>
```

Comment:
Nice work wrapping the footer inside the footer tags :+1: however the footer doesn't look like the original kindly improve on the footer.

### ORIGINAL
![footer Imp orig](https://user-images.githubusercontent.com/32361963/91546675-7e4c0c00-e92b-11ea-8299-4cc3f87ad8df.jpg)
### CLONE
![footer imp clone](https://user-images.githubusercontent.com/32361963/91546714-8f951880-e92b-11ea-880a-fb4a49104f08.jpg)